slatedb/src/db/builder.rs

```diff
@@ -48,14 +48,14 @@
 //! ```
 //! use slatedb::{Db, Error};
 //! use slatedb::object_store::memory::InMemory;
-//! use slatedb::db_cache::moka::MokaCache;
+//! use slatedb::db_cache::foyer::FoyerCache;
```

Comment:
Noticed this was failing without `--all-features` since we moved to Foyer by default.